app/src/main/java/com/codesoom/assignment/controllers/ProductController.java

```diff
@@ -1,55 +1,101 @@
 package com.codesoom.assignment.controllers;
 
+import com.codesoom.assignment.application.AuthenticationService;
 import com.codesoom.assignment.application.ProductService;
 import com.codesoom.assignment.domain.Product;
 import com.codesoom.assignment.dto.ProductData;
+import com.codesoom.assignment.utils.JwtUtil;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
 import javax.validation.Valid;
 import java.util.List;
 
+/**
+ * 상품에 대한 요청을 처리합니다.
+ */
 @RestController
 @RequestMapping("/products")
 @CrossOrigin
 public class ProductController {
     private final ProductService productService;
+    private final AuthenticationService authenticationService;
 
-    public ProductController(ProductService productService) {
+    public ProductController(ProductService productService, AuthenticationService authenticationService) {
         this.productService = productService;
+        this.authenticationService = authenticationService;
     }
 
+    /**
+     * 상품 전체 목록에 대한 요청을 처리합니다.
```

Comment:
JavaDoc을 작성할 때에는 리턴값에 대한 설명에 집중해 보세요. 때로는 심플한 설명이 더 이해하기 쉽습니다.
```suggestion
     * 상품 전체 목록을 리턴합니다.
```

Reply:
처리한다는 단어는 정확히 어떤 걸 할지 설명하지는 못했군요. 서비스와 주석이 같아질 것 같아 저렇게 표현했지만, 명확하게 설명하는게 더 중요하다는 걸 놓쳤었네요.